### PR TITLE
update the quench values for oatmeal and grits

### LINKED ISSUE #68313: oatmeal has no quench
### Is your feature request related to a problem? Please describe.

Oatmeal (and other similar dishes such as grits, boiled grains, etc) are boiled in water but don’t have a quench value. Meanwhile pasta dishes like spaghetti do have a quench value even though most of that water is strained off after cooking.

### Solution you would like.

Someone could measure how much water remains after cooking oatmeal, but I figure it has to be at least three quarters, if not more.

### Describe alternatives you have considered.

We could just copy the quench value from the spaghetti recipe; it’d be better than literally zero.

### Additional context

```json
  {
    "type": "COMESTIBLE",
    "id": "spaghetti_cooked",
    "name": { "str_sp": "boiled noodles" },
    "weight": "148 g",
    "color": "yellow",
    "spoils_in": "20 hours",
    "container": "box_small",
    "comestible_type": "FOOD",
    "symbol": "%",
    "quench": 12,
    "calories": 222,
    "description": "Fresh wet noodles.  Fairly bland, but fills you up.",
    "price": 20,
    "price_postapoc": 50,
    "material": [ "wheat" ],
    "volume": "250 ml",
    "flags": [ "EATEN_HOT", "TRADER_AVOID" ],
    "vitamins": [ [ "calcium", 1 ], [ "iron", 10 ] ]
  },
  {
    "type": "COMESTIBLE",
    "id": "oatmeal_cooked",
    "name": { "str_sp": "cooked oatmeal" },
    "weight": "247 g",
    "color": "light_gray",
    "spoils_in": "10 days",
    "comestible_type": "FOOD",
    "symbol": "%",
    "calories": 151,
    "description": "A filling and nutritious New England classic that has sustained pioneers and captains of industry alike.",
    "price": 500,
    "price_postapoc": 50,
    "material": [ "wheat" ],
    "volume": "250 ml",
    "flags": [ "EATEN_HOT", "NUTRIENT_OVERRIDE" ],
    "fun": 2,
    "vitamins": [ [ "calcium", 1 ], [ "iron", 10 ] ]
  }
```

### PR DESCRIPTION
**Pull request by db48x**
#### Summary
Balance "update the quench values for oatmeal and grits"

#### Purpose of change

Avoid wasting water when cooking oatmeal.

#### Describe the solution

@Papaya-Plan added quench values for oatmeal when he fixed #68313, copying the values from pasta. But when you cook pasta you drain off most of the water, which is very different from oatmeal.

I therefore weighed some oatmeal before and after heating it up. It lost 20 grams out of the 220 grams that I had added to it, meaning it retained 90.45% of the water used to cook it. 50 × 90% = 45.

Also, the mass of the grits was wrong because it left out the mass of the absorbed water. I just assume it’s the same as oatmeal.

#### Describe alternatives you've considered

None; I don’t think there’s any other way to change these values.

#### Testing
Screenshot from 2023-10-03 00-19-46

### CODE REVIEW

**Comment by Calavera20012** [comment]
Since most food values are not based in water % (if I'm remembering correctly, I could be wrong) giving these food products what ,essentially ammounts to "how much it quenchs your thirst" similar to those of water when soda is considerably lower (despite a full-calorie soda or carbonated water being typically composed of 90% water, while diet soda is about 99% water​). I think that this, without other changes to food based on water content and possibly looking at the Thirst need, would break the game's balancing a bit.

**Comment by db48x** [comment]
You could be right, though I’ll note that oatmeal and grits don’t normally have enough salt or sugar to make you more thirsty. Meanwhile a soda is about 10% sugar by volume, so of course it doesn’t really quench your thirst very well.

File: data/json/items/comestibles/wheat.json [hunk 1]
```diff
@@ -250,13 +252,14 @@
     "type": "COMESTIBLE",
     "id": "cheese_grits",
     "name": { "str_sp": "cheese grits" },
-    "weight": "80 g",
+    "weight": "306 g",
```
Comment (Maleclypse) [inline data/json/items/comestibles/wheat.json:255]:
```suggestion
    "weight": "276 g",
```

Comment (Maleclypse) [reply data/json/items/comestibles/wheat.json:255]: Trying to get the density test to pass.

**Comment by db48x** [comment]
Oops. I just pushed that commit by sheer muscle memory, before I ran the tests :)

**Pull request merged**